db/seeds.rb

```diff
@@ -2,87 +2,127 @@ class Seed
+  def generate_orders
+    car_count = Car.count
+
+    User.all.each_with_index do |user, i|
+      order = Order.create!(user_id: user.id)
+      car = Car.find(rand(1..car_count))
+
+      order.order_items.create!( user_id: user.id,
+                                 store_id: car.store.id,
+                                 days: 2,
+                                 car_id: car.id )
+
+      puts "Order #{i}: Order for #{user.username} created!"
     end
   end
 
-  # def generate_orders
-  #   100.times do |i|
-  #     user  = User.find(Random.new.rand(1..50))
-  #     order = Order.create!(user_id: user.id)
-  #     add_items(order)
-  #     puts "Order #{i}: Order for #{user.name} created!"
-  #   end
-  # end
-  #
-  # private
-  #
-  # def add_items(order)
-  #   10.times do |i|
-  #     item = Item.find(Random.new.rand(1..500))
-  #     order.items << item
-  #     puts "#{i}: Added item #{item.name} to order #{order.id}."
-  #   end
-  # end
+  private
+
+    def generate_andrews_store
+      user = User.create!(username: "andrew@turing.io",
+                          password: "password")
+
+      user.roles.create!(name: "store_admin")
+
+      store = Store.create!(name: "#{Faker::Company.name} Cars")
+      store.users << user
+
+      puts "Store 20 (#{store.name}) created for Andrew! =O"
+    end
+
+    def car_images
+      [ "https://images.unsplash.com/photo-1444598664632-8403d0237760?crop=entropy&dpr=0.67&fit=crop&fm=jpg&h=1175&ixjsv=2.1.0&ixlib=rb-0.3.5&q=80&w=2175",
+        "https://images.unsplash.com/photo-1441148345475-03a2e82f9719?crop=entropy&dpr=0.67&fit=crop&fm=jpg&h=1175&ixjsv=2.1.0&ixlib=rb-0.3.5&q=80&w=2175",
+        "https://images.unsplash.com/photo-1417444419095-aae558146a39?crop=entropy&dpr=0.67&fit=crop&fm=jpg&h=1175&ixjsv=2.1.0&ixlib=rb-0.3.5&q=80&w=2175",
+        "https://images.unsplash.com/12/car.jpg?crop=entropy&dpr=0.67&fit=crop&fm=jpg&h=1175&ixjsv=2.1.0&ixlib=rb-0.3.5&q=80&w=2175",
+        "https://images.unsplash.com/19/bad-ass.JPG?crop=entropy&dpr=0.67&fit=crop&fm=jpg&h=1175&ixjsv=2.1.0&ixlib=rb-0.3.5&q=80&w=2175",
+        "https://images.unsplash.com/photo-1446891549729-620bb632b9d1?crop=entropy&dpr=0.67&fit=crop&fm=jpg&h=1175&ixjsv=2.1.0&ixlib=rb-0.3.5&q=80&w=2175",
+        "https://images.unsplash.com/uploads/1413686514683468a35c4/bcccfd71?crop=entropy&dpr=0.67&fit=crop&fm=jpg&h=1175&ixjsv=2.1.0&ixlib=rb-0.3.5&q=80&w=2175",
+        "https://images.unsplash.com/photo-1445098344978-4080e28b6355?crop=entropy&dpr=0.67&fit=crop&fm=jpg&h=1175&ixjsv=2.1.0&ixlib=rb-0.3.5&q=80&w=2175" ]
+    end
+
+    def denver_neighbourhoods
+      [ Location.create(name: 'Auraria'),
+        Location.create(name: 'Baker'),
+        Location.create(name: 'Capitol Hill'),
+        Location.create(name: 'City Park'),
+        Location.create(name: 'Cheeseman Park'),
+        Location.create(name: 'Cherry Creek'),
+        Location.create(name: 'Five Points'),
+        Location.create(name: 'Highlands'),
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.